src/Product/Product.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import axios from "axios";
+import Navbar from "../Components/Navbar/Navbar-comp";
+import "./Product.css"
+import Filter from "../Filter/Filter";
+
+
+import { useState, useEffect } from "react";
+
+function Product() {
+  const [product, setProduct] = useState([]);
+
+  async function getData() {
+    const result = await axios.get("/api/products");
+    console.log(result);
```

Comment:
You could remove the unnecessary `console.log`'s from the PR. Those are helpful for debugging but don't look good on a hosted site.